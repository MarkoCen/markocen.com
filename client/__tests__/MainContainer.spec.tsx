import { shallow } from 'enzyme';
import React from 'react';
import MainPage from '../containers/Main';

describe('<MainPage />', () => {
    test('should render component', () => {
        const wrapper = shallow(<MainPage />);
        expect(wrapper.find('div').length).toEqual(1);
    });
});
