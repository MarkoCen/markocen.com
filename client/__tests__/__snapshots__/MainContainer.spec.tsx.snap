// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainPage /> should render component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MainPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <styled.h1>
        菩提本无树
        <br />
        明镜亦非台
        <br />
        本来无一物
        <br />
        何处惹尘埃
        <br />
      </styled.h1>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          "菩提本无树",
          <br />,
          "明镜亦非台",
          <br />,
          "本来无一物",
          <br />,
          "何处惹尘埃",
          <br />,
        ],
      },
      "ref": null,
      "rendered": Array [
        "菩提本无树",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        "明镜亦非台",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        "本来无一物",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        "何处惹尘埃",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
      ],
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <styled.h1>
          菩提本无树
          <br />
          明镜亦非台
          <br />
          本来无一物
          <br />
          何处惹尘埃
          <br />
        </styled.h1>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            "菩提本无树",
            <br />,
            "明镜亦非台",
            <br />,
            "本来无一物",
            <br />,
            "何处惹尘埃",
            <br />,
          ],
        },
        "ref": null,
        "rendered": Array [
          "菩提本无树",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "明镜亦非台",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "本来无一物",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "何处惹尘埃",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
